samples/snippets/csharp/VS_Snippets_CLR/conceptual.parallel.for/cs/for1.cs

```diff
@@ -6,11 +6,10 @@
 
 public class Example
 {
-   public static void Main()
+   public static void Main(string[] args)
    {
       long totalSize = 0;
-
-      String[] args = Environment.GetCommandLineArgs();
+      
       if (args.Length == 1) {
```

Comment:
This `if` logic in now wrong, as the [`Environment.GetCommandLineArgs()`](https://docs.microsoft.com/en-us/dotnet/api/system.environment.getcommandlineargs?view=net-5.0) automatically adds the first element to the array as the "The first element is the executable file name".

Reply:
Thanks @IEvangelist. I have committed the changes.